Introspection is a hard requirement to build gnome-desktop-testing upstream, but
it isn't needed anymore.

Upstream-Status: Submitted
Signed-off-by: Ross Burton <ross.burton@intel.com>

diff --git a/configure.ac b/configure.ac
index 7460c69..6249caf 100644
--- a/configure.ac
+++ b/configure.ac
@@ -27,8 +27,6 @@ AC_SUBST(WARN_CFLAGS)
 LT_PREREQ([2.2.4])
 LT_INIT([disable-static])
 
-GOBJECT_INTROSPECTION_REQUIRE([1.34.0])
-
 PKG_CHECK_MODULES(GIO_UNIX, [gio-unix-2.0 >= 2.34.0])
 GIO_UNIX_CFLAGS="$GIO_UNIX_CFLAGS -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_36 -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_36"
 
